dj_cqrs/mixins.py

```diff
@@ -404,3 +408,54 @@ def cqrs_delete(cls, master_data):
             raise NotImplementedError
 
         return cls.cqrs.delete_instance(master_data)
+
+    @staticmethod
+    def should_retry_cqrs(current_retry, exception=None):
+        """Checks if we should retry the message after current attempt.
+
+        :param current_retry: Current number of message retries.
+        :type current_retry: int
+        :param exception: Exception instance raised during message consume.
+        :type exception: Exception, optional
+        :return: True if message should be retried, False otherwise.
+        :rtype: bool
+        """
+        default_max_retries = 10
```

Comment:
There is a file: `dj_cqrs/constants.py`
Please, consider to place `default_max_retries` and `default_retry_delay ` (below) constants there.